src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserFactory.java

```diff
@@ -0,0 +1,53 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import lombok.Builder;
+import lombok.Getter;
+
+public final class ForgottenPasswordUserFactory {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)